src/common/ActivityInfo/ActivityInfo.tsx

```diff
@@ -79,6 +101,31 @@ export const ActivityInfo: React.FC<InfoProps> = (props: InfoProps) => {
                 ? "🙄 I'm no longer interested"
                 : "🤔 Mark me interested"}
             </Button> */}
+            <Dropdown>
+              <Dropdown.Toggle id="add-to-calendar">
+                Add to calendar
+              </Dropdown.Toggle>
+              <Dropdown.Menu>
+                <Dropdown.Item
+                  target="_blank"
+                  href={
+                    `https://www.google.com/calendar/render?` +
+                    `action=TEMPLATE&` +
+                    `text=${props.title}&` +
+                    `dates=${formatGoogleTime(
+                      props.startTime,
+                      props.endTime
+                    )}&` +
+                    `details=Attend Here: ${props.mediaLink}&` +
```

Comment:
This renders the Zoom link directly which is the temporary link for now (```https://zoom.us/join/zoomRoomIdGoesHere```).
- Can we just link to the Portal event page? That way we can track the click eventually.
- Also, Can you do a conditional render and hide this button if the event has already passed?